dune-project

```diff
@@ -1,26 +1,28 @@
 (lang dune 3.0)
 
 (using menhir 2.1)
-
+(using dune_site 0.1)
 (using mdx 0.2)
+(using directory-targets 0.1)
 
 (cram enable)
 
+(generate_opam_files true)
+
 (name owi)
 
 (license AGPL-3.0-or-later)
 
 (authors
  "Léo Andrès <contact@ndrs.fr>"
- "Pierre Chambart <pierre.chambart@ocamlpro.com>")
+ "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+ "Filipe Marques")
```

Comment:
cc @filipeom, I believe you quite deserved to be listed as an author now :smile: 

If you want to add an e-mail there, you can either tell me which one, either add it after this get merged.